cli/src/address_lookup_table.rs

```diff
@@ -113,15 +121,18 @@ impl AddressLookupTableSubCommands for App<'_, '_> {
                                 .takes_value(true)
                                 .required(true)
                                 .validator(is_pubkey)
-                                .help("Address of the lookup table")
+                                .help("Address of the lookup table"),
                         )
                         .arg(
                             Arg::with_name("authority")
                                 .long("authority")
                                 .value_name("AUTHORITY_SIGNER")
                                 .takes_value(true)
                                 .validator(is_valid_signer)
-                                .help("Lookup table authority [default: the default configured keypair]")
+                                .help(
+                                    "Lookup table authority [default: the default configured \
+                                     keypair]",
```

Comment:
Nit: In some of the other ones, you put
```
[default: the default configured keypair]
```
on its' own line. Altho the first line `Lookup table authority` is pretty bare so I'm fine with the wrap too

Reply:
Dang it! Thanks for catching these.